### DISCUSSION TITLE
CanCloseDialog and back button

### DISCUSSION THREAD

**Discussion by stesvis** [PrismLibrary/Prism#2401] [Xamarin.Forms]
Hello, is it possible to prevent the back button in Android from closing a dialog?

I tested it with `return false;` hardcoded, like this:
```cs
        public bool CanCloseDialog()
        {
            return false;
        }
```

But in Android the back button still closes the dialog. Thanks.

I tried this too in code behind but i didn't work:
```cs
protected override bool OnBackButtonPressed()
{
    return true;
}
```

**Comment by dansiegel** [comment]
This was recently added and is available on Sponsor Connect in the latest CI builds.

**Reply by stesvis** [reply]
Hello thank you I've never heard of it. Is there a link that explains a bit?

**Reply by dansiegel** [reply]
@stesvis Sponsor Connect is a site that I've built for people who finally help to support my open source work. The Prism CI builds are available on there to anyone who is an active Sponsor for myself or @brianlagunas. 
https://github.com/sponsors/dansiegel
